src/Jackett.Common/Definitions/torrentmasters.yml

```diff
@@ -0,0 +1,151 @@
+---
+id: torrentmasters
+name: TorrentMasters
+description: "TorrentMasters is a Hungarian private tracker for MOVIES / TV / GENERAL"
+language: hu-HU
+type: private
+encoding: UTF-8
+links:
+  - https://torrentmasters.info/
+
+caps:
+  categorymappings:
+    - {id: 92, cat: Books, desc: "e-Book (ENG)"}
+    - {id: 78, cat: Books, desc: "e-Book (HUN)"}
+    - {id: 80, cat: Other, desc: "Egyéb"}
+    - {id: 85, cat: Movies/DVD, desc: "Film - DVD/9 (ENG)"}
+    - {id: 66, cat: Movies/DVD, desc: "Film - DVD/9 (HUN)"}
+    - {id: 84, cat: Movies/HD, desc: "Film - HD (ENG)"}
+    - {id: 68, cat: Movies/HD, desc: "Film - HD (HUN)"}
+    - {id: 82, cat: Movies, desc: "Film - XviD (ENG)"}
+    - {id: 64, cat: Movies, desc: "Film - XviD (HUN)"}
+    - {id: 72, cat: PC/Games, desc: "Játék - ISO"}
+    - {id: 73, cat: Console, desc: "Játék - Konzol"}
+    - {id: 94, cat: PC, desc: "Játék - RIP"}
+    - {id: 79, cat: PC/Mobile-Other, desc: "Mobil/PDA"}
+    - {id: 95, cat: PC, desc: "Programok - Egyéb (Win)"}
+    - {id: 74, cat: PC/ISO, desc: "Programok - ISO (Win)"}
+    - {id: 75, cat: PC/Mac, desc: "Programok - Linux/Unix/Mac OS"}
+    - {id: 10, cat: TV, desc: "Sorozat - DVD (ENG)"}
+    - {id: 10, cat: TV, desc: "Sorozat - DVD (HUN)"}
+    - {id: 89, cat: TV/HD, desc: "Sorozat - HD (ENG)"}
+    - {id: 97, cat: TV/HD, desc: "Sorozat - HD (HUN)"}
+    - {id: 86, cat: TV, desc: "Sorozat - XviD (ENG)"}
+    - {id: 90, cat: TV, desc: "Sorozat - XviD (HUN)"}
+    - {id: 98, cat: XXX, desc: "XXX - HD"}
+    - {id: 99, cat: XXX/DVD, desc: "XXX - DVD"}
+    - {id: 76, cat: XXX/ImageSet, desc: "XXX - Képek"}
+    - {id: 69, cat: XXX/XviD, desc: "XXX - XviD"}
+    - {id: 70, cat: Audio, desc: "Zene - HUN"}
+    - {id: 71, cat: Audio, desc: "Zene - Külföld"}
+
+  modes:
+    search: [q]
+
+settings:
+  - name: username
+    type: text
+    label: Username
+  - name: password
+    type: password
+    label: Password
+
+  - name: sort
+    type: select
+    label: Sort requested from site
+    default: 0
+    options:
+      0: added
+      1: name
+      3: comments
+      5: size
+      6: grabbed
+      7: seeders
+      8: leechers
+
+  - name: type
+    type: select
+    label: Order requested from site
+    default: desc
+    options:
+      desc: desc
+      asc: asc
+
+login:
+  method: post
+  path: takelogin.php
+  inputs:
+    username: "{{ .Config.username }}"
+    password: "{{ .Config.password }}"
+
+  test:
+    selector: a[href^="logout.php"]
+
+search:
+  paths:
+    - path: letoltes.php
+  inputs:
+    # doesn't seem to support selecting multiple categories
+    # if multiple categories are selected only one of them will be used
+    $raw: "{{ range .Categories }}cat={{.}}&{{end}}"
+    search: "{{ .Keywords }}"
```

Comment:
cat=x usually only supports single category.
try `    $raw: "{{ range .Categories }}c{{.}}=1&{{end}}"`

Reply:
That doesn't seem to work on this site.

Reply:
OK, in which case hard code a cat=0. Thanks.

Reply:
Okay, but wouldn't it be better if it worked for at least one category?

Reply:
No. If someone wanted to search for a term in all Movie categories, but the site only does so for the first category, you end up with fewer results than if you just search all categories and Jackett filters the results.

Reply:
Alright, cat = 0

Reply:
Although, this would prevent users from just listing (without a search string) things in a rare category such as 'PC/Mac' since usually there won't be any torrents with these categories among the first 100.

Reply:
That's preferable to completely breaking apps like Sonarr et al.